server/__init__.py

```diff
@@ -87,16 +86,6 @@ def validateSettings(event):
         event.preventDefault().stopPropagation()
 
 
-def onGetItem(event):
-    User = ModelImporter.model('user', 'isic_archive')
-    itemResponse = event.info['returnVal']
-
-    # Hide the 'originalFilename' metadata on Images from non-site admins
-    if 'originalFilename' in itemResponse.get('meta', {}):
-        if not User.canReviewDataset(getCurrentUser()):
-            del itemResponse['meta']['originalFilename']
```

Comment:
Is there a database migration in conjunction with this PR that removes `meta.originalFilename` from existing images?